src/boot/boot_main.c

```diff
@@ -15,9 +16,15 @@ void bootproc(void) {
     StackCheck_Init(&sBootThreadInfo, sBootThreadStack, STACK_TOP(sBootThreadStack), 0, -1, "boot");
 
     osMemSize = osGetMemSize();
+#if PLATFORM_N64
+    func_80001720();
+    bootclear();
+    osInitialize();
+#else
     bootclear();
     __osInitialize_common();
     __osInitialize_autodetect();
+#endif
```

Comment:
`osInitialize` is a macro that expands two those two functions since [libultra K](https://github.com/decompals/ultralib/blob/52d323da2e464d5ed0d0766295761555b7995fa1/include/PR/os_host.h#L121)
```
#define osInitialize()           \
{                                \
    __osInitialize_common();     \
    __osInitialize_autodetect(); \
}
```

```suggestion
#if PLATFORM_N64
    func_80001720();
#endif
    bootclear();
    osInitialize();
```

Reply:
but we don't have this macro in OoT, and I don't want to deal with libultra now :cold_sweat:

Reply:
I think it would be beneficial to introduce this macro. Maybe plonk it on functions.h like you did with `osInitialize`.

```c
#if PLATFORM_N64
void osInitialize(void);
#else
#define osInitialize()           \
{                                \
    __osInitialize_common();     \
    __osInitialize_autodetect(); \
}
#endif
```

We can remove it when we incorporate ultralib